doc/contributing.md

```diff
@@ -0,0 +1,35 @@
+### Typechecking and tests
+
+Run all tests, including typechecking with mypy, with:
+
+```
+inv test
+```
+
+
+### Changing dependencies
+
+It's critical to update the lockfile every time dependencies are changed. Whenever you
+update the `environment.yml`, please update the lockfile with:
+
+```
+conda-lock -p linux-64 -p osx-64
```

Comment:
Someone on GitHub [recently showed](https://github.com/conda/conda-lock/issues/442#issuecomment-1608302554) me a better way to do this! In `environment.yml`, you can have a platforms key, and then you only need to run `conda-lock` with no arguments. Much nicer in my opinion :)

```suggestion
conda-lock
```

- [ ] specify platforms list in environment.yml instead of command line